x-pack/test/functional/page_objects/endpoint_page.ts

```diff
@@ -8,16 +8,33 @@ import { FtrProviderContext } from '../ftr_provider_context';
 
 export function EndpointPageProvider({ getService }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
 
   return {
     async welcomeEndpointMessage() {
       return await testSubjects.getVisibleText('welcomeEndpointMessage');
     },
-    async navigateEndpointsPage() {
-      await testSubjects.click('menuEndpoints');
+    async navToEndpointList() {
+      const sideNav = await testSubjects.find('endpointSideNav');
+      const navItems = await sideNav.findAllByCssSelector('button');
+      const endpointButton = navItems[3];
+      await endpointButton.click();
     },
-    async endpointsSearchBar() {
-      return await testSubjects.find('endpointsSearchBar');
+    async searchForObject(objectName: string) {
+      const searchBox = await testSubjects.find('endpointsSearchBar');
+      await searchBox.clearValue();
+      await searchBox.type(objectName);
+      await searchBox.pressKeys(browser.keys.ENTER);
+    },
+    async getSavedObjectsInTable() {
+      const table = await testSubjects.find('endpointsListTable');
+      const cells = await table.findAllByCssSelector('td:nth-child(3)');
```

Comment:
I think, if you are hardcoding an array index value, you should make a comment about what you're expecting that 3rd item to actually be.

Reply:
It might be worth creating a page object that can retrieve data from a table based on column name and row.  We would be able to reuse to validate data in other table types and for the other columns in the table.  As well you get the reference by column name and row number so a bit more readable.